activation/atxdb.go

```diff
@@ -186,11 +186,14 @@ func (db *DB) ProcessAtx(ctx context.Context, atx *types.ActivationTx) error {
 		log.FieldNamed("atx_node_id", atx.NodeID),
 		atx.PubLayerID)
 	if err := db.ContextuallyValidateAtx(atx.ActivationTxHeader); err != nil {
-		db.log.WithContext(ctx).With().Error("atx failed contextual validation",
+		db.log.WithContext(ctx).With().Warning("atx failed contextual validation",
 			atx.ID(),
 			log.FieldNamed("atx_node_id", atx.NodeID),
 			log.Err(err))
 		// TODO: Blacklist this miner
+		// NOTE(nkryuchkov): Sometimes, this case may be seen in devnet logs
+		// with error: "last atx is not the one referenced",
+		// perhaps we need to add a threshold for blacklisting.
```

Comment:
it is very likely due to the bug in the codebase https://github.com/spacemeshos/go-spacemesh/issues/2871 . i don't think that these todo's about blacklisting should be implemented until we know that software always works as intended.

right now blacklisting will do more harm than good. i would just remove todo and note, i think it is more confusing then useful

Reply:
I think removing only this to-do won't give any benefits as we have similar to-do's in the codebase. Maybe something like `TODO: Consider blacklisting after we ensure that this case always works as intended (ref: #2871)`?

Reply:
i don't think that this todo make any sense. you can't just go and fix this todo, it needs to be designed properly for the whole project. in my opinion you just added more unnecessary comments to an already confusing todo, that's why i suggested to remove it altogether

Reply:
> you can't just go and fix this todo, it needs to be designed properly for the whole project
Well, that's true. I'll remove it then